python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md

```diff
@@ -1,4 +1,4 @@
-name: Initializing & Cleaning a Dataset
+name: Preparing a Dataset
 
 description: Learn what Series and DataFrames are, and how to initialize and prepare a dataset for analysis.
```

Comment:
Can we abbreviate the description too?

Would something like this work?

```suggestion
description: Learn how to use Series and DataFrames to prepare a dataset for analysis.
```

Reply:
```suggestion
description: Learn what Series and Dataframes are, and how to prepare a dataset for analysis.
```
Maybe like this?

Reply:
Looks good to me